function.peer.event.php

```diff
@@ -50,13 +50,18 @@ function peer_event() {
 		// No Existing Peer
 		!$peer ||
 		// IP has changed.
-		$peer['ip'] != $_GET['ip'] ||
+		$peer['ipv4'] != $_GET['ipv4'] ||
+		$peer['ipv6'] != $_GET['ipv6'] ||
 		// Port has changed.
-		$peer['port'] != $_GET['port'] ||
+		$peer['portv4'] != $_GET['portv4'] ||
+		$peer['portv6'] != $_GET['portv6'] ||
 		// check whether seeding status match
 		$peer['state'] != $settings['seeding']
 	) {
 		require_once __DIR__.'/function.peer.new.php';
+		require_once __DIR__.'/function.peer.delete.php';
+		// Delete and reload the peer
+		peer_delete();
```

Comment:
I kept getting duplicate rows; REPLACE was not being honored, as such, the same peer was being returned multiple times, and confusing transmission. I'll revert this part of the commit, but I don't trust it to be right or work properly.

Here's my version of MySQL:
mysql  Ver 14.14 Distrib 5.5.44, for debian-linux-gnu (x86_64) using readline 6.3